src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Main{
+    public static void main(String[] args) {
```

Comment:
The method 'main' should not have an empty line at the beginning of its definition.